--- tree.orig/stage1/01-sys-tweaks/files/fstab	2022-05-31 10:21:03.000000000 +0000
+++ tree/stage1/01-sys-tweaks/files/fstab	2022-07-08 14:02:54.000000000 +0000
@@ -1,3 +1,5 @@
 proc            /proc           proc    defaults          0       0
 BOOTDEV  /boot           vfat    defaults,flush    0       2
 ROOTDEV  /               ext4    defaults,noatime  0       1
+DATADEV  /data           exfat   umask=0002,uid=1000,gid=33,x-systemd.device-timeout=3min  0       0
+/data/docker.fs /data/docker    ext4    defaults,noatime,x-systemd.requires=/data       0       0
