--- tree.orig/stage2/01-sys-tweaks/01-run.sh	2022-05-31 10:21:03.000000000 +0000
+++ tree/stage2/01-sys-tweaks/01-run.sh	2022-07-25 14:57:50.000000000 +0000
@@ -11,9 +11,13 @@
 
 install -m 755 files/rc.local		"${ROOTFS_DIR}/etc/"
 
+install -m 755 files/offspot.yaml  "${ROOTFS_DIR}/boot/firmware/"
+
+install -m 755 files/offspot_bi_init_resize.sh "${ROOTFS_DIR}/usr/sbin/"
+
 if [ -n "${PUBKEY_SSH_FIRST_USER}" ]; then
 	install -v -m 0700 -o 1000 -g 1000 -d "${ROOTFS_DIR}"/home/"${FIRST_USER_NAME}"/.ssh
-	echo "${PUBKEY_SSH_FIRST_USER}" >"${ROOTFS_DIR}"/home/"${FIRST_USER_NAME}"/.ssh/authorized_keys
+	echo -e "${PUBKEY_SSH_FIRST_USER}" >"${ROOTFS_DIR}"/home/"${FIRST_USER_NAME}"/.ssh/authorized_keys
 	chown 1000:1000 "${ROOTFS_DIR}"/home/"${FIRST_USER_NAME}"/.ssh/authorized_keys
 	chmod 0600 "${ROOTFS_DIR}"/home/"${FIRST_USER_NAME}"/.ssh/authorized_keys
 fi
@@ -47,6 +52,13 @@
 systemctl enable resize2fs_once
 EOF
 fi
+on_chroot << EOF
+systemctl disable resize2fs_once
+EOF
+
+on_chroot << EOF
+ln -s /usr/bin/vim.tiny /usr/local/bin/vim
+EOF
 
 on_chroot <<EOF
 for GRP in input spi i2c gpio; do
